src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -7,6 +7,7 @@
 import threading
 import time
 import re
+import requests
```

Comment:
let's use lazy loading of modules
import it only for the function that needs it